Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-list/SubmissionsList.tsx

```diff
@@ -112,6 +113,13 @@ const SubmissionsList = ({
                             />
                         </div>
                     </div>
+                    <ExecutionResult
```

Comment:
![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/f477d6cd-fa9b-4e87-a77f-8d45a68f2792)
Elements position is off.

Reply:
This is in the SubmissionGridRow.tsx component, but still a valid point. I fixed it.